exercises/transpose/example.py

```diff
@@ -0,0 +1,10 @@
+def transpose(input_lines):
+    lines = input_lines.split("\n")
+    zipped = map(list,
+                 [line.ljust(len(max(lines, key=len)))
+                  for line in lines])
+    return "\n".join("".join(line) for line in zip(*zipped)).strip()
+
+
+if __name__ == '__main__':
```

Comment:
Sorry I missed this the last time, but we don't need that here and the call would even throw an exception.